src/main/java/com/example/online_shop_api/Repository/OrderProductRepository.java

```diff
@@ -14,7 +15,7 @@ public interface OrderProductRepository extends JpaRepository<OrderProduct, Long
     List<OrderProduct> findAllByOrderId(@Param("orderId")Long orderId);
     //[OrderProduct(id=null, order=null, product=Product(id=1, name=name1, price=2.10, quantity=0, imageLocation=null), quantity=3),
```

Comment:
we dont need commented lines

Reply:
The commented lines are an example of what result this method gives. 
it is needed for this particular repo since **List<OrderProduct> findAllByOrderId** and **List<OrderProduct> findAllByOrder_Id** give different results.